src/EmailPage/index.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+import { Typography } from '@kiwicom/orbit-components';
+import image from '../../static/mailbox@2x.png';
+
+type Props = {
+  email: string,
+};
+
+const style = css`
+  .Email {
+    width: 480px;
+    padding-top: 240px;
+  }
+  p.title {
+    width: 305px;
+    height: 34px;
+    font-size: 28px;
+    font-weight: bold;
+    line-height: 1.2;
+    margin-bottom: 8x;
```

Comment:
:scream:  8XXXXX
Also, shouldn't it be `6` instead of `8` based on zeplin? 
https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5aa6b8a48e193557a2446de1

Reply:
🤦‍♂️